### PR TITLE
Contribution of Additional Tenable_io Commands - 020821

### PR DESCRIPTION
**Pull request by rod-castillo**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/34554

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Demisto
- [ ] 5.0.0
- [ ] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by ShirleyDenkberg** [comment]
@barchen1 @guykeller 
The following files were edited:
- integration yml
- Release Notes
- Integration README
The description file had no information.  I spoke with Rod and suggested that there be information on how to obtain the API key and the Secret key. 
There was no Pack README file to add content to.

**Review by guykeller — CHANGES_REQUESTED** [review]
A more general question - did we verify we don't get any duplicates after the fix?

File: Packs/Tenable_io/Integrations/Tenable_io/Tenable_io.py [hunk 1]
```diff
@@ -397,10 +492,87 @@ def get_scan_status_command():
...
         'Status': scan_details['info']['status']
     }
-    return get_entry_for_object('Scan status for {}'.format(scan_id), 'TenableIO.Scan(val.Id === obj.Id)', scan_status)
+    return get_entry_for_object('Scan status for {}'.format(scan_id), 'TenableIO.Scan(val.Id && val.Id === obj.Id)', scan_status)
+
+
+def is_valid_scan_id(scan_id):
+    if scan_id == "" or scan_id.isalpha():
+        return False
+    return True
```
[the remark this answers is not in the document]
Comment (barchen1) [reply Packs/Tenable_io/Integrations/Tenable_io/Tenable_io.py:501]: @rod-castillo wanted to have an input validation to make sure that scan_id is a number

File: Packs/Tenable_io/Integrations/Tenable_io/Tenable_io.py [hunk 2]
```diff
@@ -496,18 +496,16 @@ def get_scan_status_command():
 
 
 def is_valid_scan_id(scan_id):
-    if scan_id == "" or scan_id.isalpha():
-        return False
-    return True
+    return not (scan_id == "" or scan_id.isalpha())
```
Comment (guykeller) [inline Packs/Tenable_io/Integrations/Tenable_io/Tenable_io.py:499]: @barchen1 If you want to verify this is a number, `isalpha()` won't work, you need `isdigit()`
image

Comment (guykeller) [inline Packs/Tenable_io/Integrations/Tenable_io/Tenable_io.py:499]:
```suggestion
    return not (scan_id == "" or not scan_id.isdigit())
```

**Review by guykeller — APPROVED** [review]
Last few comments and we're done from my side
Just have a last look to make sure you're confident about merging

**Review by barchen1 — APPROVED** [review]

**Pull request merged**